lib/chef/resource.rb

```diff
@@ -1090,25 +1099,23 @@ def should_skip?(action)
     # === Returns
     # <Chef::Resource>:: returns the proper Chef::Resource class
     def self.resource_for_node(short_name, node)
-      require 'chef/resource_resolver'
```

Comment:
i think i did that deliberately so that you could require this file without slurping in the resource_resolver which slurps in all of the resources.